src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -56,14 +82,14 @@ var _ webhook.Defaulter = &Cluster{}
 
 func allResourceFields(c *Cluster) []resourceField {
 	return []resourceField{
-		{c.Spec.Resources, field.NewPath("spec").Child("resources")},
+		{&c.Spec.Resources, field.NewPath("spec").Child("resources")},
```

Comment:
Could this change be moved to previous commit? https://github.com/vectorizedio/redpanda/pull/1948/commits/884ad5d8bf82e034c04cc01eb044cdb51f4d6238